miden-lib/asm/miden/tx.masm

```diff
@@ -163,17 +163,17 @@ end
 
 #! Executes the provided procedure against the foreign account.
 #!
-#! WARNING: the procedure to be invoked can not have more than 12 inputs and it can not return more
-#! than 12 elements back. Otherwise exceeding elements will not be provided to the procedure and
+#! WARNING: the procedure to be invoked can not have more than 15 inputs and it can not return more
+#! than 15 elements back. Otherwise exceeding elements will not be provided to the procedure and
```

Comment:
Is it 15 or 16?

Reply:
Should be 15, since we use `dyncall` to execute the provided procedure, so it's 15 similar to other places